src/sentry/api/serializers/models/commit.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
+
+from sentry.api.serializers import Serializer, register
+from sentry.models import Commit
+
+
+@register(Commit)
+class CommitSerializer(Serializer):
+    def serialize(self, obj, attrs, user):
+        return {
+            'id': obj.key,
+            'message': obj.message,
+            'author': {
+                'name': obj.author.name,
+                'email': obj.author.email,
+            } if obj.author else None,
+            'dateCreated': obj.date_added,
```

Comment:
We uniformly name this everywhere. The db column has always been date_added, and we expose it as dateCreated. The API does not need to reflect how our database operates, and thus they're named different (as are a lot of other things).